src/forms/FormFieldAdvancedSelection/AdvancedSelection.stories.tsx

```diff
@@ -206,6 +209,17 @@ export const KitchenSink = (): ReactElement => {
 						createNewOption
 					}
 				},
+
+				{
+					name: "selectLimitOfOptions",
+					label: "Advanced selection with selectLimit prop",

```

Comment:
I'd suggest adding this at the end so users know what to expect.
```suggestion
					label: "Advanced selection with selectLimit prop (Max 2 options)",

```